app.py

```diff
@@ -230,6 +233,18 @@ def submit_form(name_):
     return form, 201
 
 
+@app.route('/import', methods=['POST'])
+@cross_origin(headers=['Content-Type', 'Authorization'])
+def import_data():
```

Comment:
This is the new web hook url. It makes two calls to the CMS to get the latest data, and then imports that data into the database.